src/05-derivatives/05-functional-derivatives.md

```diff
@@ -0,0 +1,390 @@
+# Functional derivatives
+
+## Functional derivatives maps
+
+A derivative map is a measure derived from a functional series, mapped onto spatial
+locations as defined by voxels in a volume or vertices on a surface.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<label>/
+        func/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.{nii[.gz],gii,dscalar.nii}
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.json
+```
+
+for example:
+
+```Text
+pipeline1/
+    sub-001/
+        func/
+            sub-001_task-rest_run-1_space-MNI305_mean.nii.gz
+            sub-001_task-rest_run-1_space-MNI305_mean.json
+```
+
+The following table lists allowed suffixes and their corresponding measures:
+
+| `<suffix>`       | Measure                                                          |
+| ---------------- | ---------------------------------------------------------------- |
+| `mean`           | Mean across the temporal/4th dimension of the data               |
+| `std`            | Standard deviation across the temporal/4th dimension of the data |
+| `tsnr`           | Temporal SNR (i.e. mean / std)                                   |
+| `sfs`            | Signal fluctuation sensitivity                                   |
+| `alff`           | Amplitude low frequency fluctuations                             |
+| `falff`          | Fractional amplitude of low frequency fluctuations               |
+| `reho`           | Regional homogeneity (voxelwise only)                            |
+| `dcb`, `dcw`     | Voxelwise degree centrality binary and weighted                  |
+| `ecb`, `ecw`     | Voxelwise eigenvector centrality binary and weighted             |
+| `lfcdb`, `lfcdw` | Local functional connectivity density                            |
+| `vmhc`           | Voxel mirrored homotopic connectivity                            |
```

Comment:
These should become `stat-<label>`s for consistency with BEP 16 and BEP 29. We need a new suffix. I'm a little inclined to `boldmap`, which makes clear that it's a statistic based on a bold series. @Remi-Gau @cmaumet @tsalo (and anybody else) any opinions?

Reply:
And the more general rule for the derivative suffix would be `{rawsuffix}map` given that functional data has several potential suffixes for raw?

https://github.com/bids-standard/bids-specification/blob/6d7eb0fa8e40e481440976a353bb02d451935db5/src/schema/rules/files/raw/func.yaml#L5C1-L6C12

Reply:
That would work for me. On the other hand, since it's not a quantitative map, not sure if it's violating an implicit principle started by qmri.